publish-mvn.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -euo pipefail
+
+###
+# Usage: ./publish-mvn.sh <release-zip>
+#
+# Publishes the content of a release bundle ZIP file to Maven Central. This
+# script expects the following environment variables to be set to appropriate
+# values (which can be achieved by using scripts/with-signing-key.sh):
+# + GNUPGHOME       - A GnuPG home directory containing the signing key
+# + KEY_ID          - The ID of the GnuPG key that will be used for signing
+# + KEY_PASSWPHRASE - The passphrase of the provided key.
+###
+
+if [ $# -ne 1 ]; then
+    echo "Missing release zip file argument"
+    echo "Usage: ./publish-mvn.sh <release-zip>"
```

Comment:
On the CodeBuild project, this thing is going to work from a directory (the current one, most likely).

Can we make it an option to do either zip or directory? Or something to enable that use case.

Reply:
Yeah, it should "discover" the zip file from the current directory. That's how the codebuild setup will work